lib/src/components/NxVulnerabilityDetails/__tests__/DetailLink.test.tsx

```diff
@@ -23,6 +23,6 @@ describe('DetailLink', function() {
     expect(shallowRender).toHaveProp('href', 'http://foo.bar');
     expect(shallowRender).toHaveProp('target', '_blank');
     expect(shallowRender).toHaveProp('rel', 'noreferrer');
-    expect(shallowRender).toHaveText('Label');
+    expect(shallowRender).toHaveText('Label<NxFontAwesomeIcon />');
   });
```

Comment:
Should really have an additional test checking what kind of icon it is

Reply:
> Should really have an additional test checking what kind of icon it is

https://github.com/sonatype/sonatype-react-shared-components/pull/129/commits/4954fcc98780510b3dd1bcd8e74d5a018bbdade3